client/src/store/modules/i18n/en.js

```diff
@@ -84,6 +84,43 @@ export default {
     FACEBOOK_ICON: 'f',
     SCHEDULIA: 'Schedulia',
     USER_MENU_BUTTON: 'My Account',
+
+    MENU: 'Menu',
+    MENU_HOME: 'Home',
+    MENU_EDIT_EVENT_TYPE: 'Edit event type',
+
+    CONFIRM_DATE: 'Confirm',
+    SELECT_DATE_AND_TIME: 'Select a date and time',
+
+    DURATION_MIN: 'min',
+
+    WEEK_DAYS: ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'],
+    MONTHS: [
```

Comment:
when you defined a constant for each month, it was better than array. Because, the localization must reflect the word or phrase it contains

Reply:
fixed these problems and changed components layout data according to data that is required based on db scheme as there is missing layout data on mocks